giraph.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'giraph/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'giraph'
+  spec.version       = Giraph::VERSION
+  spec.authors       = ['Erman Celen']
+  spec.email         = ['erman@upserve.com']
+
+  spec.summary       = 'Composable GraphQL for micro-services'
+  spec.description   = 'Expose a remote GraphQL endpoint within another as a regular field'
+  spec.homepage      = 'https://github.com/upserve/giraph'
+  spec.license       = 'MIT'
+
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir        = 'bin'
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.require_paths = ['lib']
+
+  spec.required_ruby_version = '~> 2.1'
+
+  spec.add_runtime_dependency 'graphql', '~> 0.15.3'
```

Comment:
I have _okay_ test coverage for the behavior of the `Subquery` class which is the most-heavily-reliant class, however, much more would be warranted in this case. Almost all tests actually go through GraphQL internals (e.g full query execution) so even if not every possibility is not covered, we will know if the integration breaks in a major way due to changes in `graphql-ruby` internals.

Having coverage directly for `graphql-ruby`s internal APIs that I'm using and ensure the current interface stays intact is certainly a good idea. Will do.